tests/vmlifecycle_test.go

```diff
@@ -159,6 +159,32 @@ var _ = Describe("Vmlifecycle", func() {
 			})
 		})
 
+		Context("with boot order", func() {
+			It("should be able to boot from 'middle' disk", func() {
```

Comment:
If you let that test run twice, once booting cirros and once booting alpine by changing the boot priority this should be sufficient. You can use `table.DescribeTable` to create a test table.

Reply:
Incoming...